providers/plugin.rb

```diff
@@ -20,14 +20,16 @@
 #
 
 action :create do
-  directory '/etc/td-agent/plugin' do
+  d = directory '/etc/td-agent/plugin' do
     owner "root"
     group "root"
     mode "0755"
     action :create
   end
 
-  remote_file "/etc/td-agent/plugin/#{new_resource.plugin_name}.rb" do
+  new_resource.updated_by_last_action(true) if d.updated_by_last_action?
```

Comment:
This new check is intended?

Reply:
Yes.
I want updated_by_last_action flag to be set true if either one needs to converge.